js/deer-render.js

```diff
@@ -204,6 +204,255 @@ DEER.TEMPLATES.shadow = (obj, options = {}) => {
     }
 }
 
+DEER.TEMPLATES.statusComment = (obj, options = {}) => {
+    if (!obj['@id']) { return null }
+    //We know who made the comment in obj.comment.value.author
+    let commentElem = 
+    `<div style="text-transform: none;" title="${obj.comment.value.text}" > 
+        <span> See status comment from <deer-view deer-template="label" deer-id="${obj.comment.value.author}">  </deer-view> </span>
+        <div class="statusDrawer text-dark is-hidden">           
+            <pre>${obj.comment.value.text}</pre>
+        </div>
+     </div>   `
+
+    return {
+        html: commentElem,
+        then: (elem) => {
+            if(obj.comment.value.text){
+                elem.classList.remove("is-hidden")
+            }
+            elem.setAttribute(DEER.SOURCE, obj.comment.source.citationSource)
+            elem.addEventListener("click", e => {
+                if(e.target.tagName === "PRE"){
+                    //This way they can highlight and select text
+                    return
+                }
+                if(elem.querySelector(".statusDrawer").classList.contains("is-hidden")){
+                    elem.querySelector(".statusDrawer").classList.remove("is-hidden")
+                }
+                else{
+                    elem.querySelector(".statusDrawer").classList.add("is-hidden")
+                }
+            })
+        }
+    }
+}
+
+DEER.TEMPLATES.reviewedStatus = (obj, options = {}) => {
+    if (!obj['@id']) { return null }
+    // This Annotation is continually overwritten, so no history wildcard necessary.
+    const query =
+    {
+        "body.moderate.releasedTo" : "http://store.rerum.io/v1/id/61ae693050c86821e60b5d13",
```

Comment:
Almost. The Motivation of the Annotation is `moderating`, but I don't think we need to bury `releasedTo` inside `moderate`

Reply:
Yeah I know, I wanted this to stick out like a sore thumb.  I need an example of one of those Annotations so I know what to put here.